docs/UserGuide.md

```diff
@@ -10,6 +10,65 @@ MediBase3 (MB3) is a **desktop app for doctors to manage their patients and appo
 * Table of Contents
 {:toc}
 
+## How to use this User Guide
+Our user guide is tailored to ensure you can fully unlock the potential of MediBase3, whether you're new to MediBase3 or have been using the application for quite some time.
+In this section, we aim to provide you with the necessary tools to effectively navigate and utilize the guide.
+
+### Navigating the User Guide
+Each aspect of MediBase3 is split into different sections, which are accessible via the Table of Contents above.
+
+**For new MediBase3 users:**
+1. We **recommend** that you finish reading this section to better understand the format of the user guide.
+1. Once you are done, do head to the [Quick Start](#quick-start) section to get started with setting up MediBase3.
+
+**For experienced MediBase3 users:**
+1. If you need a quick refresher on how to use a specific feature in MediBase3, you can navigate to the [Features](#features) section.
+1. You may also skip to the [Command Summary](#command-summary) section for a quick overview of all the commands available in MediBase3 and their formats.
+
+For any additional information or queries, you can refer to the [FAQ](#faq) section or the [Known Issues](#known-issues) section.
+
+### Glossary
+As you read the user guide, you might encounter some unfamiliar technical terms. The table below provides the definitions for the key terms we will be using throughout the guide.
+
+Term | Definition
+---- | ----------
+Command |An input that is given to the application to perform a specific action. Commands are typically entered via the command box in the application.
+CLI | Command Line Interface(CLI) is a type of text-based interface that is used to interact with software via commands.
+GUI | Graphical User Interface(GUI) is a type of interface that allows users to interact with electronic devices through graphical icons and visual indicators.
+JAR | Java ARchive(JAR) is a package file format typically used to aggregate many Java class files and associated metadata and resources into one file for distribution.
+JSON | JavaScript Object Notation(JSON) is a lightweight data-interchange format that is easy for humans to read and write and easy for machines to parse and generate.
+Parameter | A parameter refers to a field that requires input from the user. For example, in the command `delete i/S1234567A`, `i/S1234567A` is a parameter.
+Prefix | A prefix is a keyword that is used to identify the type of parameter that follows it. For example, in the command `delete i/S1234567A`, `i/` is the prefix for the NRIC parameter.
```

Comment:
Do you want to add another point for `preamble`?

Reply:
Sure, I will update this in my next PR for the UG 👍🏻